src/lib/screenshot-helper.js

```diff
@@ -0,0 +1,23 @@
+const booleanHelper = require('./boolean-helper');
+
+const screenshotHelper = {
+    shouldTakeScreenshot: (status) => {
+        return booleanHelper.isTruthy(process.env['chimp.captureAllStepScreenshots']) ||
+            (
+                status !== 'passed' &&
```

Comment:
With the **retries** functionality of Mocha, when it fails, the status is I think either an empty string or pending. Until it actually fails all retries, then it should be changed to failed. I'm not sure we should be taking screenshots when the test is still pending or it could still be retried.